SR2024/2024-03-15-competition-details.md

```diff
@@ -0,0 +1,63 @@
+---
+to: Student Robotics 2024
+subject: SR2024 Competition Details
+attachment: https://drive.google.com/file/d/1JQG_IKOtnvEUPSob1XWTma55FHRK_r4o/view
+---
+
+Hello everyone!
+
+It's just under a month until the big day - [the competition][competition-event-page]!
+
+Before then, we have a few important updates and reminders for you:
+
+## Attendance details
+
+Thanks to everyone who has already provided their attendance details. If you haven't done that yet, or things have changed substantially please [provide attendance details](https://forms.gle/8YhfdRbfDWP3KKNy8).
+
+Doors open on both days at 9am and each day will finish around 5:30pm. The event will formally start at 10:30am on Saturday. There are now full day schedules on the [event page][competition-event-page]. **If you are expecting to arrive after 10am on Saturday, please let us know.**
+
+You'll have access to your pits and the test arena when you arrive, giving your team time to work on their robot before things get going in earnest.
+
+The event is open to the public, so you're welcome to bring family and friends as guests, however please note that space in the pits is limited.
+
+We'll be publishing rules for behaviour in/around the competition venue when you arrive. If you have any questions or concerns beforehand, please let us know.
```

Comment:
Reads wierdly, sounds like we'll be publishing when you arrive.

Reply:
Perhaps "providing ... when you arrive"? I agree the current framing is weird (though copied from last year). We don't seem to put these on the website, though I think they are in the booklets, so it's not clear that any publishing happens in the moment of their arrival.

Reply:
116d48cd2f825e199a76dbd723653bdd7c7bc0bc

Reply:
Perhaps something like "you'll find rules for behaviour in/around the competition venue...in your booklet. Please have a read and let us know if you have any questions or concerns" would address this, although it didn't jump out as a big issue to me.

Reply:
Yeah, I believe it was the booklets which the publishing was referring to. The teams don't get those until the day of the event, at which point I doubt they or we will really be in a position to suggest/make changes.